tree/10_ncf_internals/modules/packages/apt_get

```diff
@@ -193,7 +193,7 @@ def list_updates(online):
         #                repository(ies)      arch (might be optional)
         #                       |               |
         #                    /--+-\   /---------+---------\
-                         "\s+[^[)]*(\s\[(?P<arch>[^]\s]+)\])?\).*", line)
+                         "(?:\s+\S+)*?(\s+\[(?P<arch>[^]\s]+)\])?\).*", line)
```

Comment:
Do you know why it changed ?

Reply:
ENT-3438: make apt_get package module work with repositories containing spaces in the label

Reply:
I will backport this to 4.1 as well